### PR TITLE
Update eppo guide for Next.js SSR integration

### PR DESCRIPTION
**Pull request by greghuels**
Updated the SSR documentation with a more-straightforward approach. Some of the changes were automatic by VS Code, but it shouldn't hurt anything.

### CODE REVIEW

File: docs/guides/engineering/nextjs-setup.md [hunk 1]
```diff
@@ -9,14 +9,16 @@ This guide walks through how to setup an Eppo Feature Flag on a client and serve
...
 Provide the following answers when prompted:
+
 - What is your project named? `react-sdk-walkthrough`
 - Would you like to use TypeScript? `Yes`
 - Would you like to use ESLint? `Yes`
 - Would you like to use Tailwind CSS? `No`
 - Would you like to use `src/` directory? `No`
 - Would you like to use App Router? (recommended) `Yes`
-- Would you like to customize the default import alias (@/*)? `No`
+- Would you like to customize the default import alias (@/\*)? `No`
```
Comment (greghuels) [inline docs/guides/engineering/nextjs-setup.md:21]: The backslash was added automatically as an escape character by VS code formatting

File: docs/guides/engineering/nextjs-setup.md [hunk 2]
```diff
@@ -225,54 +233,113 @@ export async function getAssignment(flagKey: string, subjectKey: string): Promis
...
+
+export default async function getEppoClient() {
+  return await eppoClient;
+}
 ```
 
-### Update main page
+### String assignment helper
+
+Let's also create a helper function for fetching string assignments. This helper will ensure that the Eppo SDK is initialized before returning a value. To do this, create a new `app/shared/get-string-assignment.ts` file, and add the code snippet below.
```
Comment (aarsilv) [inline docs/guides/engineering/nextjs-setup.md:263]: ❤️

**Review by aarsilv — APPROVED** [review]
Thanks for sprucing this up 📈

**Review by hhargreaveseppo — APPROVED** [review]
lgtm

**Pull request merged**